### DISCUSSION TITLE
Guardian Steam Cannon is missing statuseffect <remove />

### DISCUSSION THREAD

**Discussion by Draconis13** [FakeFishGames/Barotrauma#14123] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Whenever a Guardian gets onboard/above a submarine and it has a guardiansteamcannon item and is killed it drops the steam cannon and if it gets in the submarine the item cannot be picked up and continues to rattle around the submarine forever as it is missing the following in <repairtool> 

      <StatusEffect type="OnNotContained" target="This">
        <Remove />
      </StatusEffect>

### Reproduction steps

guardian leaves ruin and dies in such a way that the steam cannon drops inside a hull of the submarine. 

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/ba87b5198cfc71156d2d58ccb4c3de9f752c865f (commit in our private repo). Coming up in a future unstable build.
